Automattic/woocommerce-payments
# Display a Confirmation Modal before enabling Test Mode

Fixes #7846 

#### Changes proposed in this Pull Request
- Adds a confirmation modal when the merchant is enabling Test Mode.
- 
#### Testing instructions
- Mock the values of  `useTestMode` and `useDevMode` (preferably in `hooks.js`) to see the `Enable test mode`  checkbox on  `Payments > Settings > General`
- Check that the Confirmation Modal is displayed only when the merchant attempts to enable test mode.
- Validate the Enable, Confirm, Close buttons, content, link etc.

-------------------


**Post merge**
